packages/api-client-core/package.json

```diff
@@ -45,14 +45,14 @@
     "ws": "^8.18.2"
   },
   "devDependencies": {
-    "@types/node": "^16.11.7",
+    "@types/node": "^22.13.14",
```

Comment:
If `api-client-core` starts using an API that's only available in node 22+ then apps that upgrade their packages but are still on framework versions less than 1.4 will start to break.

Our oldest supported framework version (1.0) uses node 20. Can we make lower this dependency to v20?